dashboard/app/controllers/levels_controller.rb

```diff
@@ -174,6 +174,8 @@ def get_serialized_maze
   # GET /levels/:id/edit_blocks/:type
   # Action for using blockly workspace as a toolbox/startblock editor.
   # Expects params[:type] which can be either 'toolbox_blocks' or 'start_blocks'
+  # Javalab also uses this route to edit starter code, and sets the type param
+  # to 'start_sources'
```

Comment:
It's not just Java Lab. From what I remember, App Lab uses this route for the same purpose with the same type param. Web Lab may as well.

Reply:
Oh huh. Nevermind then. Just a quick check then (not blocking for this PR): do you think we can move to using the same route as all the other labs?

Reply:
I think we didn't want to use `start_blocks` because it was confusing for Java Lab

Reply:
Yeah I think we could use the same route -- even right now, I think you can put anything in the URL after edit_blocks and you'll get the edit start code page (eg, `/levels/123/edit_blocks/xyz`)